src/YearMonth.php

```diff
@@ -156,10 +162,10 @@ public function compareTo(YearMonth $that): int
         if ($this->year > $that->year) {
             return 1;
         }
-        if ($this->month < $that->month) {
+        if ($this->month->value < $that->month->value) {
             return -1;
         }
-        if ($this->month > $that->month) {
+        if ($this->month->value > $that->month->value) {
```

Comment:
Should there be a `compareTo` method on `Month` ?

Reply:
I'm not sure if it makes sense to compare months without a year, I'd leave it like this for now!